### ISSUE TITLE
Add a new uplink policy to have maximum video quality independently from the number of remote attendees

### ISSUE THREAD

**Issue by OasisLiveForever** [aws/amazon-chime-sdk-js#1527]
In my application I have an instructor that makes fitness classes with users (e.g. Yoga) and I want users to have the best video quality of the instructor,

In the pull request #1491 https://github.com/aws/amazon-chime-sdk-js/pull/1491, NScale uplink policy has been updated to scale down resolution based on number of remote videos.
If I upgrade to version 2.16.0, I think I will have a lower quality for the instructor video as the number of users grows.

Is there a way to set the maximum quality for the instructor video without scaling it down as the number of users grows?

I think it would be very useful to add a MaximumQualityUplinkPolicy to the SDK.

**Comment by ltrung** [comment]
@OasisLiveForever Thanks for asking. In #1491, we do not exactly change the existing behavior of limit the video quality when the number of attendees increase. Previously, we only reduce the max bit rate and #1491 just make sure we also adjust the resolution. However, one thing to note is that the default policy always keep the active speaker's video quality and does not limit max bit rate or scale down resolution.

Regarding your use case, the instructor should be the active speaker right? If the default policy works for you before, then it should still work. 
You can also manually call [setHasBandwidthPriority](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/videouplinkbandwidthpolicy/NScaleVideoUplinkBandwidthPolicy.ts#L154) to manually set the instructor to be of high priority and thus would never have his/her video quality reduced.

**Comment by OasisLiveForever** [comment]
@ltrung Yes, in my case the instructor is typically the active speaker but also users can speak.
I'm using simulcast, does setHasBandwidthPriority work also with simulcast uplink policy?

Many thanks.

**Comment by ltrung** [comment]
Because simulcast sends 2 video layer in high and low already, it does not implement `setHasBandwidthPriority`. `setHasBandwidthPriority` is only for the default NScale uplink bandwidth policy. 
You can use priority-based downlink policy to always priority the instructor and set the target size to high for the instructor.
